src/Migrations/2022_03_28_073146_create_topic_table.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Uasoft\Badaso\Module\LMSModule\Helpers\DatabaseHelper;
+
+class CreateTopicTable extends Migration
+{
+    private $tableName;
+
+    public function __construct()
+    {
+        $this->tableName = DatabaseHelper::getBadasoTableName('topic');
+    }
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create($this->tableName, function (Blueprint $table) {
+            $table->id();
+            $table->text('title');
```

Comment:
I guess this one is supposed to be varchar instead of text. It's not convenient to see a damn-long topic isn't it?
ref: https://dbdiagram.io/d/620f0ccb485e433543d3d036